docs/case_coverage.md

```diff
@@ -0,0 +1,31 @@
+# Coverage of use cases and key attributes
+
+Coverage of each of the uses cases and key atributes
+defined in:
+ * [use cases](https://github.com/nodejs/benchmarking/blob/master/docs/use_cases.md)
+ * [key attributes](https://github.com/nodejs/benchmarking/blob/master/docs/runtime_attributes.md)
+
+# Use Cases
+
+Use case                                   |Benchmark with coverage| Notes
+-------------------------------------------|-----------------------|-------
+Service oriented architectures (SOA) | |
+Microservice-based applications | |
+Generating/serving dynamic web page content | Acme Air Ops/s <BR>Acme Air Latency |
+SPA applications with bidirectional communication over WebSockets and/or HTTP/2 | |
```

Comment:
It would be expand out SPA. I haven't seen it used too frequently (but maybe my viewfinder is too narrow?)